app/views/shared/views/ems_common/_form.html.haml

```diff
@@ -28,7 +28,7 @@
         - else
           = h(@edit[:ems_types][@edit[:new][:emstype]])
     = render(:partial => "form_fields", :locals => {:url => url})
-    - if @edit[:new][:emstype] && @edit[:new][:emstype] == 'rhevm'
+    - if @edit[:new][:emstype] && %w(rhevm openstack_infra).include?(@edit[:new][:emstype])
```

Comment:
Would prefer to ask the model about whether they are allowed to do things such as set the Keystone API port.